{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Not affected",
      "package_name": "ipa",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Not affected",
      "package_name": "ipa",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "idm:DL1/ipa",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    }
  ],
  "threat_severity": "Low",
  "public_date": "2020-04-09T03:31:00Z",
  "bugzilla": {
    "description": "CVE-2020-1703 ipa: failure to invalidate session after password change",
    "id": "1793049",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1793049"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "3.8",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:L/I:L/A:N",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-613",
  "statement": "",
  "acknowledgement": "This issue was discovered by Pritam Singh (Red Hat).",
  "name": "CVE-2020-1703",
  "document_distribution": "",
  "details": [
    "No description is available for this CVE."
  ],
  "references": null
}